src/commands/deploy/deploy.ts

```diff
@@ -88,6 +95,48 @@ async function deploy(actionContext: IActionContext, arg1: vscode.Uri | string |
     await notifyDeployComplete(context, node, context.workspaceFolder.uri.fsPath);
 }
 
+async function tryToUpdateWorkerProcessTo64BitIfRequired(context: IDeployContext, siteConfig: WebSiteManagementModels.SiteConfigResource, node: SlotTreeItemBase, language: ProjectLanguage): Promise<void> {
+    try {
+        const functionProject: string | undefined = await tryGetFunctionProjectRoot(context.workspaceFolder.uri.fsPath);
+        if (functionProject === undefined) {
+            throw new Error(localize('failedToFindFuncHost', 'Unable to locate Azure Functions project.'));
+        }
+        const projectFiles: string[] = await dotnetUtils.getProjFiles(language, functionProject);
+        if (projectFiles.length === 0) {
+            throw new Error(localize('unableToFindProj', 'Unable to detect project file.'));
+        }
+        const mainProject: string = path.join(functionProject, projectFiles[0]);
+        const platformTarget: string | undefined = await dotnetUtils.tryGetPlatformTarget(mainProject);
+        if (platformTarget === 'x64' && siteConfig.use32BitWorkerProcess === true) {
+            const message: string = localize('overwriteSetting', 'Detected 64 bit project. Update setting in Azure Portal?');
```

Comment:
How about this:

> The remote app targets "{0}", but your local project targets "{1}". Update remote app to "{1}"? This will restart the app.

We are updating the setting for them, so they don't have to use the portal like you said. Also, I want them to know we will restart the app.

Reply:
Changed